apache-rat/pom.xml

```diff
@@ -117,6 +117,46 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>CLI help generation</id>
+            <phase>pre-site</phase>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <executable>java</executable>
+          <outputFile>${project.basedir}/src/site/apt/cli_help.txt</outputFile>
```

Comment:
Should we add this file to .gitignore as well? as it is freshly generated in each build.